buttercup.el

```diff
@@ -1701,3 +1701,8 @@ With buttercup minor mode active the following is activated:
 
 (provide 'buttercup)
 ;;; buttercup.el ends here
```

Comment:
This comment should mean that the file ends here, and not that it is then followed by further comments.

Reply:
Updated.